config/environments/development.rb

```diff
@@ -32,10 +32,22 @@
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
```

Comment:
why you changed it?

Reply:
I wanted to see the errors when I worked with email. I forgot to set it `false` again.